sdcflows/interfaces/fmap.py

```diff
@@ -663,28 +663,19 @@ def au2rads(in_file, newpath=None):
     """Convert the input phase difference map in arbitrary units (a.u.) to rads."""
     from scipy.stats import mode
     im = nb.load(in_file)
-    data = im.get_fdata(dtype='float32')
+    data = im.get_fdata(caching='unchanged')  # Read as float64 for safety
```

Comment:
I'm extremely comfortable using double memory for one volume to keep things deterministic.